modules/creating-a-machine-pool-ocm.adoc

```diff
@@ -86,6 +86,10 @@ endif::openshift-rosa[]
 .. Expand the *Edit node labels and taints* menu.
 .. Under *Node labels*, add *Key* and *Value* entries for your node labels.
 .. Under *Taints*, add *Key* and *Value* entries for your taints.
+[NOTE]
+====
+Creating a machine pool with taints is only possible if the cluster already has at least one machine pool without a taint.
+====
 .. For each taint, select an *Effect* from the drop-down menu. Available options include `NoSchedule`, `PreferNoSchedule`, and `NoExecute`.
```

Comment:
Above highlight message make this line wrapped as another charactered list which starts with `a` again.